shopping-cart/app/shoppingcart/common/ExtendedLineItemBean.java

```diff
@@ -0,0 +1,36 @@
+package shoppingcart.common;
+
+import common.contexts.UserContext;
+import common.controllers.ReverseRouter;
+import common.models.AttributeBean;
+import common.models.LineItemBean;
+import common.models.ProductDataConfig;
+import io.sphere.sdk.carts.LineItem;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class ExtendedLineItemBean extends LineItemBean {
```

Comment:
Extended has more info, which is not required for a simplified version of the cart (i.e. mini-cart). Right now that means it additionally has attributes, but it can have more info in the future (i.e. availability, shipping info...). I cannot figure out any other name. I'm also not sure what to document, I can only say it has more info, but I guess that's evident by the name. Can you give suggestions?